autotst/conformer/systematic.py

```diff
@@ -174,7 +174,7 @@ def opt_conf(i, rmsd_cutoff):
             label = conformer.smiles
             type = 'species'
 
-        if isinstance(calc, ase.calculators.calculator.FileIOCalculator):
+        if isinstance(calculator, ase.calculators.calculator.FileIOCalculator):
```

Comment:
Thanks for the catch. This function was designed to reference the ASE calculator attached to the conformer object preceding this line (line 161). However, a typo was making this reference a different calculator (line 264). Both should result in the same outcome, but this is now correct.